internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -64,6 +64,18 @@
     (OPTIONAL) Whether to install and configure fluentd to forward log events to the collector (default: $true)
     .EXAMPLE
     .\install.ps1 -access_token "ACCESSTOKEN" -with_fluentd $false
+.PARAMETER with_dotnet_tracing
```

Comment:
Consider, with_dotnet_instrumentation.
The change should be propagated to other places like `dotnet_tracing_msi_url`, etc.

Reply:
I like this idea. The thing is that the msi is called signalfx-dotnet-tracing-xyz.msi, so I ended up using this terminology everywhere. Will fix.

Reply:
Having second thoughts, since mixing dotnet_instrumentation and dotnet_tracing (e.g. https://github.com/signalfx/signalfx-dotnet-tracing) might be confusing.

Reply:
We are keeping the name `signalfx-dotnet-tracing` because of legacy but it will deliver other features like metrics and Always On Profiling. In light of that I think it is better to use the more general `instrumentation` instead.

Reply:
Nice idea. I think we can punt on this until we bring in the other languages though. Since we're generating the script args, I don't think backwards compatibility is going to be a problem.